README.md

```diff
@@ -104,4 +104,4 @@ This utility runs on Linux, Mac OS, and Windows.
 Firmware updates are supported through the
 [fwupd](https://github.com/fwupd/fwupd/) project, and are distributed using the
 related Linux Vendor Firmware Service. Settings are stored on EEPROM and are
-maintained through firmware updates.
+maintained through firmware updates.
```

Comment:
The last line is showing up as changed because the newline at the end of the file has been removed. Text files should ideally end with a newline (otherwise things like `cat` will end up including the terminal prompt immediately after on the same line as the content, etc). I tried to fix it for you, but I don't have access to push to your branch. Can you either allow maintainers access to push to your branch, or add the newline back?